setup.sh

```diff
@@ -62,18 +99,59 @@ EOF
 }
 
 setup_compose() {
+  echo "** Creating Redash Docker compose file **"
+
   cd "$REDASH_BASE_PATH"
   GIT_BRANCH="${REDASH_BRANCH:-master}" # Default branch/version to master if not specified in REDASH_BRANCH env var
-  curl -OL https://raw.githubusercontent.com/getredash/setup/"$GIT_BRANCH"/data/docker-compose.yml
-  echo "export COMPOSE_PROJECT_NAME=redash" >>~/.profile
-  echo "export COMPOSE_FILE=$REDASH_BASE_PATH/docker-compose.yml" >>~/.profile
+
+### Remove this before committing to master
+GIT_BRANCH=debian_non_sudo_v1
+
+  curl -fsSOL https://raw.githubusercontent.com/getredash/setup/"$GIT_BRANCH"/data/docker-compose.yml
+  curl -fsSOL https://raw.githubusercontent.com/getredash/setup/"$GIT_BRANCH"/redash_make_default.sh
+  chmod +x redash_make_default.sh
   export COMPOSE_PROJECT_NAME=redash
   export COMPOSE_FILE="$REDASH_BASE_PATH"/docker-compose.yml
-  sudo docker-compose run --rm server create_db
-  sudo docker-compose up -d
+
+  echo "** Initialising fresh Redash database **"
+  docker compose run --rm server create_db
+
+  echo
+  echo "*********************"
+  echo "** Starting Redash **"
+  echo "*********************"
+  docker compose up -d
```

Comment:
Change service `data/docker-compose-up.service` to use `docker compose` too?

https://github.com/getredash/setup/blob/69e01877fe1a065d53e1018cb607256a54ed9ff7/data/docker-compose-up.service#L9

Reply:
Oops, didn't even think to look at that file.  Will take a proper look at it tomorrow and change that + anything else that seems relevant. :smile:

Reply:
Done. :smile: